### ISSUE TITLE
[Feature] Ability to indicate worker count per project

### ISSUE THREAD

**Issue by maxdevos49** [microsoft/playwright#27127]
I would like to define a playwright config like such:
```ts
export default defineConfig({
	// ...

	projects: [
		{
			name: 'Parallel Test',
			testMatch: '**/parallel/**/*.spec.ts',
			use: {...devices['Desktop Chrome']},
			retries: 1,
			workers: 6, // <-- This is the important part
		},
		{
			name: 'Sequence Test',
			testMatch: '**/sequence/**/*.spec.ts',
			use: {...devices['Desktop Chrome']},
			retries: 1,
			workers: 1, // <-- This is the important part
		},
	],
});
```

All my tests share the same test site. Settings I am testing affect the site globally and can not always be isolated between each test. Using `test.describe.configure({mode: "serial"})` half works but all tests after a failing test are skipped.

For example what I want in the project "Sequence Test":

```ts
// I want the describes to run in sequence and 1 at a time.
test.describe('Settings test 1', () => {
	// These could run in parallel or any order but 1 at a time is ok. Failures should not cause mass skipping
	test('test 1', async () => { /* ... */});
	test('test 2', async () => { /* ... */});
	test('test 3', async () => { /* ... */});
	test('test 4', async () => { /* ... */});
	test('test 5', async () => { /* ... */});
});

// "Settings test 2" group should not begin until after "Settings test 1" group is complete.
//  Failures in "Settings test 1" should not affect "Settings test 2".
test.describe('Settings test 2', () => {
	// These could run in parallel or any order but 1 at a time is ok. Failures should not cause mass skipping
	test('test 1', async () => { /* ... */});
	test('test 2', async () => { /* ... */});
	test('test 3', async () => { /* ... */});
	test('test 4', async () => { /* ... */});
	test('test 5', async () => { /* ... */});
});
```

I could alternatively run two CLI commands with specific worker count but I would like to keep the reports unified.

**Comment by dgozman** [comment]
Duplicate of #21970.
